### PR TITLE
Fix vulnerabilities found by npm audit

### PR DESCRIPTION
**Pull request by cezaraugusto**
1. `npm run test-security` should pass
2. CI should be ✅

### CODE REVIEW

**Comment by bsclifton** [comment]
Unfortunately, storybook build fails with this. It seems to be passing on master, although there is the security error. I ran into the same problem too when trying https://github.com/brave/brave-core/pull/4998 and https://github.com/brave/brave-browser/pull/8788

**Comment by cezaraugusto** [comment]
Rebased, working locally. Waiting Travis

**Comment by bsclifton** [comment]
Fixed the pep8 lint issue (my bad there, from https://github.com/brave/brave-core/pull/4947) and updated sync `DEPS` entries to use latest vulnerability free version (https://github.com/brave/sync/commits/master). Together with https://github.com/brave/brave-browser/pull/8788, this should fix everything! Building locally after rebasing

**Comment by bsclifton** [comment]
Jenkins passes completely, travis-ci only fails on `npm run build-storybook`... but it works great locally

**Comment by bsclifton** [comment]
Reproduced locally on Ubuntu 18 - will look at fix for Storybook. The `brave-browser` fix seems to work though

File: package.json [hunk 1]
```diff
@@ -291,7 +291,9 @@
     "@types/storybook__addon-centered": "^3.3.2",
     "@types/storybook__addon-knobs": "^5.0.2",
     "@types/storybook__react": "^4.0.2",
-    "babel-loader": "^8.0.6",
+    "acorn": "^7.1.1",
+    "babel-loader": "^8.1.0",
+    "babel-register": "^6.26.0",
```
Comment (cezaraugusto) [inline package.json]: I don't think we need this. Locally just `acorn` and updating `babel-loader` did the trick

Comment (bsclifton) [reply package.json]: Verified locally on Linux that it's not needed 👍 Let's let the Jenkins run finish and I can update / let Travis run / and we're good for merge 😄

**Comment by bsclifton** [comment]
CI passed completely; will do quick follow up for travis-ci (doesn't need a full CI run)

File: package.json (hunk 1 above)
Comment (bsclifton) [reply package.json]: Fixed with https://github.com/brave/brave-core/pull/5013/commits/e26d081e25dabf6c282a2425cec06f0dfe9dbde2

**Pull request merged**